src/app/games/connect-six/tests/ConnectSixMoveGenerator.spec.ts

```diff
@@ -14,25 +16,29 @@ describe('ConnectSixMoveGenerator', () => {
     const _: PlayerOrNone = PlayerOrNone.NONE;
     const O: PlayerOrNone = PlayerOrNone.ZERO;
 
+    const defaultConfig: MGPOptional<GobanConfig> = defaultGobanConfig;
+
     beforeEach(() => {
         moveGenerator = new ConnectSixMoveGenerator();
     });
-    it('should propose only one move at first turn', () => {
+
+    it('should propose only one move at first turns', () => {
         // Given the initial node
-        const width: number = ConnectSixState.WIDTH;
-        const height: number = ConnectSixState.HEIGHT;
-        const state: ConnectSixState = ConnectSixRules.get().getInitialState();
+        const width: number = defaultConfig.get().width;
+        const height: number = defaultConfig.get().height;
+        const state: ConnectSixState = ConnectSixRules.get().getInitialState(defaultGobanConfig);
```

Comment:
defaultConfig, not defaultGobanConfig icitte

Reply:
import { defaultGobanConfig } from 'src/app/jscaip/GobanConfig';
doit donc devenir
import { defaultGobanConfig as defaultConfig } from 'src/app/jscaip/GobanConfig';

Ou tu veux qu'on crée une var defaultConfig = defaultGobanConfig ?

Reply:
On a déjà defaultConfig = defaultGobanConfig, ligne 19 ! Il est d'ailleurs utilisé à la ligne 27 et 28, donc oui autant l'utiliser à la ligne 29 également